docs/pallas/tpu/collectives.rst

```diff
@@ -0,0 +1,452 @@
+Pallas Collectives
+===================
+Pallas now has support for manual DMA (between memories of the same chip) and remote DMA (between memories of different chips) support. It also has a more advanced API for nesting pipelines of compute overlapped collectives.
```

Comment:
```suggestion
Pallas has support for manual DMA (between memories of the same chip) and remote DMA (between memories of different chips) support. It also has an API for nesting pipelines of compute overlapped collectives.
```

Things like "now" in docs get stale, and it's hard to remember to update things. I'd save that language for announcements / changelog entries, and instead write docs in a 'timeless' way from the beginning.